src/styles/core/_settings.scss

```diff
@@ -1,3 +1,4 @@
+@import "variables";
```

Comment:
# This file should not be changed at all, this is for the library, not the site.